docs/standard/microservices-architecture/net-core-net-framework-containers/container-framework-choice-factors.md

```diff
@@ -43,17 +43,17 @@ There are several features of your application that affect your decision. You sh
     - Your .NET implementation choice is *.NET Framework* based on framework dependency.
     - Your container platform choice must be *Windows containers* because of the .NET Framework dependency.
 * Your application uses **SignalR services**.
-    - Your .NET implementation choice is *.NET Framework*, or *.NET Core (future release)*.
-    - Your container platform choice must be *Windows containers* because of the .NET Framework dependency.
-    - When **SignalR services** run on *.NET Core*, you can also choose *Linux containers*.
+    - Your .NET implementation choice is *.NET Framework*, or *.NET Core 2.1 or beyond*.
+    - Your container platform choice must be *Windows containers* if you chose the .NET Framework dependency.
+    - When **SignalR services** run on *.NET Core*, you can use *Linux containers or Windows Containers*.
```

Comment:
expectation of when this will happen?

Reply:
Not sure if this date is public or even confirmed, yet, so I prefer not to say it.

Reply:
It seems to be public. See https://github.com/aspnet/Home/wiki/Roadmap#210-roadmap and there's even an alpha release already: https://blogs.msdn.microsoft.com/webdev/2017/09/14/announcing-signalr-for-asp-net-core-2-0/

Reply:
Yeah, I know that SignalR ALPHA is public. What is not confirmed is when .NET Core 2.1 will be RTM. In the roadmap it says "Q1-Q2 2018"... Not sure if we want to put such a broad timeframe in the doc. I think we should just say it'll come in .NET Core 2.1. :)